### PR TITLE
Adding two Rucio projects for GSoC'21

### PR DESCRIPTION
**Pull request by mlassnig**

### CODE REVIEW

**Review by xvallspl — APPROVED** [review]

File: _gsocproposals/2021/proposal_rucio_cernbox.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * Basic understanding
+   * gRPC and Protocol Buffers
+   * Protocol Buffers
+   * HTTP REST APIs
+
+## Expected results
+
+By the end of GSoC'21 we expect at the end of the deliverable a Rucio plugin that can connect to CS3-compatible clouds, demonstrating data transfers between two endpoints in ScienceMesh.
+
+## Mentors
```
Comment (agheata) [inline _gsocproposals/2021/proposal_rucio_cernbox.md:40]: The main mentor should appear in **bold**, only in each proposal, and not in the full mentor list.

Comment (agheata) [reply _gsocproposals/2021/proposal_rucio_cernbox.md:40]: Please update also `gsoc/2021/mentors.md`

**Review by agheata — CHANGES_REQUESTED** [review]

File: _gsocproposals/2021/proposal_rucio_cernbox.md (hunk 1 above)
Comment (mlassnig) [reply _gsocproposals/2021/proposal_rucio_cernbox.md:40]: done!

File: _gsocproposals/2021/proposal_rucio_cernbox.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: Rucio and CS3APIs: enabling data management for the ScienceMesh cloud
```
Comment (aflp91) [inline _gsocproposals/2021/proposal_rucio_cernbox.md:2]: You cannot have a ':' in your title: it is confusing for jekyll…

**Review by aflp91 — CHANGES_REQUESTED** [review]

File: _gsocproposals/2021/proposal_rucio_cernbox.md (hunk 2 above)
Comment (mlassnig) [reply _gsocproposals/2021/proposal_rucio_cernbox.md:2]: o.O

**Review by aflp91 — APPROVED** [review]

**Review by agheata — APPROVED** [review]

**Pull request merged**